src/deluge/model/sample/sample_recorder.cpp

```diff
@@ -1465,8 +1467,9 @@ Error SampleRecorder::alterFile(MonitoringAction action, int32_t lshiftAmount, u
 
 		if (action != MonitoringAction::NONE || capturedTooMuch) {
 
-			FRESULT fres = f_open(&file, sample->filePath.get(), FA_WRITE);
-			if (fres) {
+			auto opened = this->file->open(sample->filePath.get(), FA_WRITE);
```

Comment:
is this not just file->open?

Reply:
it is but I'm having trouble distinguishing local `file` from instance `file` atm, so it's `this->file` until I can change it to `file_`